src/components/Case.vue

```diff
@@ -0,0 +1,169 @@
+      </svg>
+
+      <svg  v-if="status === 2 || status === 3" class="case__statusIcon" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+        <path d="M12.4557352,2.40610638 C12.7837693,1.96179563 13.4098786,1.86753482 13.8541893,2.19556897 C14.2985001,2.52360312 14.3927609,3.14971237 14.0647267,3.59402312 L6.82972843,13.3935745 L2.2873556,8.77920407 C1.8999139,8.38562133 1.9048922,7.75247592 2.29847494,7.36503422 C2.69205768,6.97759252 3.32520308,6.98257082 3.71264479,7.37615356 L6.6119057,10.3213684 L12.4557352,2.40610638 Z" fill="currentColor"></path>
+      </svg>
+
+      <svg v-if="status === 4 || status === 5" class="case__statusIcon" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
+        <path d="M10.4895924,8.5753788 L16.500272,14.586272 L22.5104076,8.5753788 C22.8708916,8.21489484 23.4381227,8.1871653 23.8304139,8.49219019 L23.9246212,8.5753788 C24.3151455,8.96590309 24.3151455,9.59906807 23.9246212,9.98959236 L23.9246212,9.98959236 L17.914272,16.000272 L23.9246212,22.0104076 C24.3151455,22.4009319 24.3151455,23.0340969 23.9246212,23.4246212 C23.5340969,23.8151455 22.9009319,23.8151455 22.5104076,23.4246212 L16.500272,17.414272 L10.4895924,23.4246212 C10.1291084,23.7851052 9.56187734,23.8128347 9.16958614,23.5078098 L9.0753788,23.4246212 C8.68485451,23.0340969 8.68485451,22.4009319 9.0753788,22.0104076 L9.0753788,22.0104076 L15.086272,16.000272 L9.0753788,9.98959236 C8.68485451,9.59906807 8.68485451,8.96590309 9.0753788,8.5753788 C9.46590309,8.18485451 10.0990681,8.18485451 10.4895924,8.5753788 Z" fill="currentColor"></path>
+      </svg>
+
+      {{ statusMessage }}
+    </div>
+  </button>
+</template>
+
+<script>
+export default {
+  name: 'Case',
+  props: {
+    caseNumber: String,
+    handleSelectCase: Function,
+    name: String,
+    status: Number,
+    statusMessage: String,
+  }
+}
+</script>
+
+<style lang="scss">
+.case {
+  position: relative;
+  display: flex;
+  flex-flow: column;
+  min-height: 5.625rem;
+  padding: 1rem 3rem 1rem 1rem;
+  width: 100%;
+  border: none;
+  border-radius: 3px;
+  cursor: pointer;
+
+  @media (min-width: 768px) {
+    padding: 0.625rem 4rem 0.625rem 0.625rem;
+  }
+
+  @media (min-width: 768px) {
+    flex-flow: row;
+  }
+
+  &:hover {
+    .case__nextIcon {
+      transform: translate(0.25rem, -50%);
+      transition: ease-in-out transform 0.3s;
+    }
+  }
+
+  &--1 {
```

Comment:
Not to necessarily a bad point, just might be nice to change to --success, --error etc. just to make styling really explicit

Reply:
also, not necessarily requiring a change here, you can just resolve if you want

Reply:
Added comments above the modifier